docs/Getting Started.md

```diff
@@ -0,0 +1,778 @@
+# Getting Started
+
+This document introduces the basic steps needed to install all components of cloudflow and deploy and run a simple cloudflow application on the GKE cluster.
+
+We will discuss the following steps in sequence:
+
+1. Download and install cloudflow CLI
+2. Go through all steps of developing a sample hello world application using cloudflow libraries
+3. Install a GKE cluster following instructions in `cloudflow-installer` repository
+4. Build the application and publish to GKE repository using `sbt buildAndPublish`
+5. Deploy the application to the cluster using cloudflow CLI
+6. Show usage of CLI helpers like `status`, `list` etc.
+7. Show how to deal with http ingress and push some data to the application
+8. Verify that the application works
+9. Undeploy the application
+
+
+## Download and install the CLI
+
+cloudflow CLI can be downloaded from [lightbend public bintray repository](https://bintray.com/lightbend/cloudflow-cli). Download the version appropriate for your platform and install it in your local system. Please make sure you have the executable in your path with proper permission settings.
+
+## Develop a sample cloudflow application
+
+In this section we will develop a sample application that will feature the main components of cloudflow. It will be a simple hello-world style application, expressive enough to demonstrate the major features of cloudflow. It will not have many of the advanced features that cloudflow offers - the main idea is to give you a feel for what it takes to build a complete application and deploy it in running condition in a GKE cluster.
+
+We will develop a simple pipeline that processes events from a wind turbine farm. The application will receive streaming data that will pass through the following transformations :
+
+* ingested by a streamlet (ingress)
+* processed by a streamlet to convert into a domain object (processor)
+* validated by a streamlet (splitter) that separates valid and invalid records
+* logged in to loggers to be checked at output (egress)
+
+Each of the above terminologies (ingress, processor, splitter and egress) are explained in the [Concepts Guide](https://blank.com). Here's an overview of the application pipeline architecture:
+
+![Application Pipeline](images/pipe.001.jpeg?raw=true "Application Pipeline")
```

Comment:
If you view how the doc is rendered there is a lot of space between the image and the net paragraph. Could you remove that?

Reply:
@skonto I tried that a lot .. looks like it's beyond my capability .. what do I need to do to remove that gap ?